### ISSUE TITLE
Missing summary description on anaconda.org

### ISSUE THREAD

**Issue by peterjc** [bioconda/bioconda-recipes#4420]
Consider three examples:

- https://bioconda.github.io/recipes/khmer/README.html
  - Summary: khmer is a library and suite of command line tools ...
- https://bioconda.github.io/recipes/pyani/README.html
  - Summary: pyani provides a package and script for calculation of genome-scale average nucleotide identity.
- https://bioconda.github.io/recipes/biopython/README.html
  - Summary: Freely available tools for computational molecular biology.
- ...

vs

- https://anaconda.org/bioconda/khmer
  - Description: *MISSING*
- https://anaconda.org/bioconda/pyani
  - Description: *MISSING*
- https://anaconda.org/bioconda/biopython
  - Description: *MISSING*
- ...

I don't think any of the BioConda packages are shown on Anaconda.org with a description.

The only descriptions I have found are the PyPI entries like https://anaconda.org/pypi/biopython which uses the ``README.rst`` file information and/or PyPI's long description metadata.

Is this something BioConda can improve in how the metadata is presented?

Is this something Anaconda.org could improve by falling back on the summary from BioConda if there is no long description?

**Comment by daler** [comment]
Hmm, seems like it's an issue on anaconda.org. The files for the biopython PyPI entry, which has the description (e.g., from https://anaconda.org/pypi/biopython/files) are tarballs of the source rather than a tarball of the recipe. That seems to be something specific to the `pypi` channel on anaconda.org.

Actually, it's not just bioconda; even the default numpy anaconda package has no description: https://anaconda.org/anaconda/numpy

We are now linting to ensure recipes have a `home:` url and a `summary:` entry; it would be nice if anaconda.org would display that on the package pages. At least the bioconda pages that we control, e.g., https://bioconda.github.io/recipes/biopython/README.html, do have the info.

**Comment by peterjc** [comment]
Yes, the pypi channel seems to be special - it was the only one I found the description being used on. Logged as https://github.com/Anaconda-Platform/support/issues/106

**Comment by CicoZhang** [comment]
I am also curious how to add the something to the area "Description"?
![image](https://user-images.githubusercontent.com/11710225/41418799-cf328574-6ff0-11e8-81d1-3549953a8238.png)

**Comment by peterjc** [comment]
Sadly the examples I picked have dated a little.

In general the ``meta.yaml`` field ``about``, ``summary:`` is (now) being used in both places:
 - Summary on https://bioconda.github.io/recipes/khmer/README.html
 - Header text on https://anaconda.org/bioconda/khmer (above the boxes, easily overlooked)

Is there a field in our ``meta.yaml`` files which would work nicely in the description box on the anaconda.org listing?

**Comment by peterjc** [comment]
I was reminded of this issue last week while helping a colleague installing from conda for the first time via the graphical client (Conda Navigator), where the description is very visible as a column in search results. Curiously, my impression was all the BioConda packages had a blank description, whereas other channels including conda-forge had text.

e.g. https://anaconda.org/conda-forge/biopython has a description, taken from the ``about:``, ``description:`` in the recipe's ``meta.yaml`` file: https://github.com/conda-forge/biopython-feedstock/blob/master/recipe/meta.yaml

However, in a counter example https://anaconda.org/conda-forge/indexed_gzip currently has no description - reflected in missing this field in https://github.com/conda-forge/indexed_gzip-feedstock/blob/master/recipe/meta.yaml

That seems to be the key - with a grep of the BioConda repository I could find some examples with the metadata which does indeed show as a description on the package page, e.g. https://anaconda.org/bioconda/khmer has no description, although
https://anaconda.org/bioconda/deeplift does.

So, can the linter bot be tweaked to start requiring the ``about:``, ``description:`` in the recipe's ``meta.yaml`` file please? That seems to be all that is needed now.

**Comment by peterjc** [comment]
(Separately, it would be ideal if Anaconda fell back on ``about:``, ``summary:`` if the ``about:``, ``description:`` field was blank)
